docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -1,25 +1,83 @@
 [role="xpack"]
 [[servicenow-sir-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow SecOps connector and action
 ++++
 <titleabbrev>ServiceNow SecOps</titleabbrev>
 ++++
 
-The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
+The {sn} SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create {sn} security incidents.
+
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading from {stack} version \<=7.15.0 to version >=7.16.0, you must complete the following within your {sn} instance before creating a new {sn} SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
+
+* Install https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] from the {sn} Store.
```

Comment:
I clicked the link and was taken to a page that said "The application that you are looking for was not fouond."

Reply:
This is expected behavior. :) The links won't be "live" until the apps are published to the ServiceNow store, meaning they won't resolve until then.